iambic/plugins/v0_1_0/aws/templates.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import annotations
```

Comment:
You shouldn't need this file anymore. I double checked and AwsConfigMixin isn't being used.